.github/workflows/deploy-ecs.yml

```diff
@@ -0,0 +1,22 @@
+name: Deploy to Dev
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        required: true
+        default: dev
+  push:
+    branches: ["main"]
+
+jobs:
+  call-workflow:
+    uses: mbta/workflows/.github/workflows/deploy-ecs.yml@v2
```

Comment:
question: do you know if dependabot will update this with our current configuration?

Reply:
I don't think so - I don't think dependabot updates things defined in these workflows (at least not as it's configured in Skate - I updated a bunch of those by hand a month or so ago).

Reply:
Well I think we should be able to have dependabot do this, but we can explore that later. https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

Reply:
[Ticket for this repo](https://app.asana.com/0/1148853526253420/1207005059291023/f)
[Ticket for Skate](https://app.asana.com/0/1148853526253420/1207005059291028/f)

Reply:
Thank you!